snap/snapcraft.yaml

```diff
@@ -816,25 +845,44 @@ parts:
 
   kuiper:
     source: https://github.com/emqx/kuiper.git
-    source-tag: 0.4.1
+    source-tag: 1.2.1-beta.1
```

Comment:
I think it's probably also worthwhile to add the kuiper update as a separate commit.

Reply:
kuiper changes to snapcraft.yaml now added as a separate commit.